### PR TITLE
Add delete notebooks section to troubleshoot

### PR DESCRIPTION
**Pull request by kelvins**
- Add a section explaining how to delete notebooks manually in the `docs/notebooks/troubleshoot.md` documentation.
- Closes https://github.com/kubeflow/website/issues/2269.

### CODE REVIEW

File: content/en/docs/notebooks/troubleshoot.md
Comment (8bitmp3) [inline content/en/docs/notebooks/troubleshoot.md:98]:
```suggestion
If you're using Google Cloud, you may encounter an error, such as:
```

**Review by 8bitmp3 — CHANGES_REQUESTED** [review]
@kelvins Hi, this PR has been hanging around for a while 😄 Do ping the team if you think something needs to be checked asap when things go stale.

Anyway, here are a few suggestions:

---

- For discoverability on Google search, maybe we can improve the title a little

```diff
- title = "Troubleshooting Guide"
+ title = "Troubleshooting guide for Kubeflow notebooks"
description = "Fixing common problems in Kubeflow notebooks"
```

---

- Line 63: let's spell out Identity-Aware Proxy (IAP) for new users and provide links to https://cloud.google.com/iap/docs/concepts-overview and https://cloud.google.com/iap/docs/enabling-kubernetes-howto (WDYT?)

```diff
- * If you are using IAP on GKE the pod will be named
+ * If you are using [Identity-Aware Proxy (IAP)](https://cloud.google.com/iap/docs/concepts-overview) [on GKE](https://cloud.google.com/iap/docs/enabling-kubernetes-howto), the pod will be named as follows:
```

---

- Use "```shell" at the beginning of each code block for improved formatting/rendering
  - Please do it across the entire doc if you can

---

- Add a brief description to `${NOTEBOOK}` after lines 22-25 even though it's self-evident (think of the users who won't understand it straight away). For example:

```diff
-     Run the commands below.		
+     Run the following commands replacing `${NOTEBOOK}` with your notebook file name:

-     ```
+     ```shell
    kubectl get notebooks -o yaml ${NOTEBOOK}
    kubectl describe notebooks ${NOTEBOOK}
    ```
    
    Check the `events` section to make sure that there are no errors.
    ...
```

---

- Check for missing blank lines (e.g. between lines 41-42 and lines 45-46)
- There's probably a couple of extra blank lines that may not be needed: line 33 and line 39, for example.

---
- Spell out Persistent Volumes Claims (PVCs) and add a link for new users (K8s has so many acronyms!)

```diff
-  First, make sure that PVCs are bounded when using Jupyter notebooks. This should		
+  First, make sure that [Persistent Volumes Claims (PVCs)](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) are bounded when using Jupyter notebooks. This should
```

---

Minor formatting:

- Line 20:

```diff
- Run the commands below.		
+ Run the following two commands lines:
```

- Line 68:
```diff
-     * Where USER@DOMAIN.EXT is the Google account used with IAP
+     * where `USER@DOMAIN.EXT` is the Google account you used with IAP
```

- Line 79:

```diff
- Once you know the name of the pod do		
+ Once you know the name of the pod, run:
```

- Line 76 (for a11y):
```diff
- * Look at the `events` to see if there are any errors trying to schedule the pod		
+ * Check the `events` for any errors trying to schedule the pod.
```

- Line 80 (present tense):
```diff
- If the error still persisted, check for the errors in the logs of containers.		
+ If the error still persists, check for the errors in the logs of containers.	
```

- Line 98:

```diff
- You may encounter error below:		
+ If you're using Google Cloud, you may encounter an error, such as:
```

---

Other matters: we have a "v1.1 out of date" alert (cc @RFMVasconcelos )

---

Thanks a lot!

**Comment by kelvins** [comment]
@8bitmp3 thanks a lot for the review and the tip :grin:

About the following comment:

> Other matters: we have a "v1.1 out of date" alert (cc @RFMVasconcelos )

Please, let me know if you guys think we can remove the "Out of date" alert.

**Comment by 8bitmp3** [comment]
/assign @StefanoFioravanzo @elikatsis @kimwnasptd @thesuperzapper
PTAL and `/approve` is all is OK

File: content/en/docs/notebooks/troubleshoot.md
Comment (RFMVasconcelos) [inline content/en/docs/notebooks/troubleshoot.md:2]: Is Kubeflow notebooks a thing? 

This may be confusing. Are we talking about Jupyter notebooks only? Or other notebook types we have available in the Kubeflow dashboard?

Comment (8bitmp3) [inline content/en/docs/notebooks/troubleshoot.md:2]:
```suggestion
title = "Troubleshooting guide for Jupyter notebooks on Kubeflow"
```

File: content/en/docs/notebooks/troubleshoot.md
Comment (8bitmp3) [inline content/en/docs/notebooks/troubleshoot.md:3]:
```suggestion
description = "Fixing common problems of Jupyter notebook deployments on Kubeflow""
```

File: content/en/docs/notebooks/troubleshoot.md
Comment (8bitmp3) [reply content/en/docs/notebooks/troubleshoot.md:2]: Good point @RFMVasconcelos. I borrowed the idea from the old description.

Check out the new suggested changes:

```
title = "Troubleshooting guide for Jupyter notebooks on Kubeflow"
description = "Fixing common problems of Jupyter notebook deployments on Kubeflow""
```

Comment (RFMVasconcelos) [reply content/en/docs/notebooks/troubleshoot.md:2]: Thank you for the suggestion @8bitmp3 ! Looking good!

**Comment by RFMVasconcelos** [comment]
@kubeflow/wg-notebook-leads can you please review and check if we can remove the "out of date" as well ?

File: content/en/docs/notebooks/troubleshoot.md
Comment (thesuperzapper) [inline content/en/docs/notebooks/troubleshoot.md:2]: I think this should stay short, especially because given context (its under the "jupyter notebook tab) its obviously about the notebooks.

I would say keep this as "Troubleshooting Guide"

File: content/en/docs/notebooks/troubleshoot.md
Comment (thesuperzapper) [inline content/en/docs/notebooks/troubleshoot.md:60]: Since these are single line  code-blocks, can you just make them inline (not triple quotes), its quite messy right now.

For example, in this case you would do:
```
* If you are using username/password auth with Jupyter the pod will be named: `jupyter-${USERNAME}`
```
Note: I referring to the non-shell ones.

**Review by thesuperzapper — CHANGES_REQUESTED** [review]
See comments.

File: content/en/docs/notebooks/troubleshoot.md
Comment (kelvins) [reply content/en/docs/notebooks/troubleshoot.md:2]: I totally agree since it is already in the "Jupyter Notebooks" section.
WDYT, @8bitmp3 @RFMVasconcelos?

File: content/en/docs/notebooks/troubleshoot.md
Comment (kelvins) [reply content/en/docs/notebooks/troubleshoot.md]: Great suggestion. I changed it to inline, can you please take another look?

File: content/en/docs/notebooks/troubleshoot.md
Comment (thesuperzapper) [reply content/en/docs/notebooks/troubleshoot.md:2]: @kelvins if you make this change and squash your commits, I will merge this right now

File: content/en/docs/notebooks/troubleshoot.md
Comment (thesuperzapper) [reply content/en/docs/notebooks/troubleshoot.md]: looks good

File: content/en/docs/notebooks/troubleshoot.md
Comment (8bitmp3) [reply content/en/docs/notebooks/troubleshoot.md:2]: The 1st reason for suggesting a longer title is because "Troubleshooting guide" may be too generic, since readers scan pages and there's a lot of Kubeflow stuff on our site. The 2nd one is to help the search engines. Having both "troubleshooting" and "Fixing common problems" in the title and the description helps with keyword search, even if the site uses Google, which is probably powered by BERT (NLP is hard, anyway).

How about we get rid of the obvious part as follows:

```suggestion
title = "Troubleshooting Guide for Jupyter Notebooks"
```

(not using sentence case here, since our titles, apparently, don't follow the Google style guide across the site :) )

Anyway, you don't have to OK my suggestions :) I'm just trying to help

Comment (thesuperzapper) [reply content/en/docs/notebooks/troubleshoot.md:2]: @kelvins I agree with @8bitmp3

**Comment by kelvins** [comment]
Hey guys, thank you so much for all the suggestions, I applied it and also did a squash on the commits. I believe the PR is ready to be merged. Can you please take another look and approve if it's all ok?

p.s. I'm not sure why the google-cla is failing, maybe because of the co-author @8bitmp3 which needs to consent?

**Comment by 8bitmp3** [comment]
@googlebot I consent.

**Comment by 8bitmp3** [comment]
/lgtm

@kelvins thanks for all the help 👍

**Comment by 8bitmp3** [comment]
approvers for `docs/notebooks/`:

/assign StefanoFioravanzo elikatsis kimwnasptd thesuperzapper

PTAL, we're waiting for your review and the `/approve` label. Thanks!

**Pull request merged**
